agent/covalent/tools/query_tool.py

```
@@ -0,0 +1,381 @@
+                return f"""❌ jq filter produced no output: {normalized_path}
+
+Call covalent_result_head again and adjust the filter to match the saved payload."""
+
+            result_size = len(formatted.encode('utf-8'))
+            execution_time = int((time.time() - start_time) * 1000)
+            
+            extracted_type = 'string'
+            try:
+                parsed = json.loads(formatted)
+                extracted_type = f'array[{len(parsed)}]' if isinstance(parsed, list) else type(parsed).__name__
+            except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)